tools/release/bumpversion.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+shopt -s globstar
+git config --global user.email "robot@canonical.com"
+git config --global user.name "Devices Certification Bot"
+git checkout -b bumpversion
+pushd $(git rev-parse --show-toplevel) > /dev/null
+cur_version=$(bumpversion --allow-dirty --dry-run --list ${1-minor} | grep current_version= | sed -r s,"^.*=",,)
```

Comment:
I'm guessing the double 'minor' default is on purpose (the first incarnation being in the workflow definition).

Reply:
double protection, but harmless.